### PR TITLE
chore: change /query/http/endpoints to /endpoints

### PR DESCRIPTION
**Pull request by zulkhair**
Closes: WORLD-754

## Overview

Remove /endpoints

## Brief Changelog

- remove /endpoints
- adjust nakama to user /world

## Testing and Verifying

manually verified by running test-game and open nakama web

### CODE REVIEW

**Comment by zulkhair** [comment]
* **#700** <a href="https://app.graphite.dev/github/pr/Argus-Labs/world-engine/700?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @zulkhair and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by technicallyty — COMMENTED** [review]
theres another occurence of query/http/endpoints in `cardinal/server/handler/endpoints.go L24`

**Review by smsunarto — APPROVED** [review]
I think I would prefer to deprecate this and serve the swagger.json file instead. It serves a similar purpose and swagger would gives us more capabilities

**Review by smsunarto — CHANGES_REQUESTED** [review]
see above comments

**Comment by zulkhair** [comment]
> I think I would prefer to deprecate this and serve the swagger.json file instead. It serves a similar purpose and swagger would gives us more capabilities

Updated
- Remove /endpoints
- Adjust nakama to get enpoint from /world

File: relay/nakama/main.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ const (
 	EnvCardinalNamespace      = "CARDINAL_NAMESPACE"
 	EnvKMSCredentialsFile     = "GCP_KMS_CREDENTIALS_FILE" // #nosec G101
 	EnvKMSKeyName             = "GCP_KMS_KEY_NAME"
-	ListEndpoints             = "query/http/endpoints"
```
Comment (smsunarto) [inline relay/nakama/main.go:30]: nit: we probably should rename it from `ListEndpoints` to `WorldEndpoint`

**Review by smsunarto — APPROVED** [review]
left optional nit, otherwise lgtm

**Pull request merged**